### ISSUE TITLE
Suggest to use `litgpt chat` after finetuning

### ISSUE THREAD

**Issue by awaelchli** [Lightning-AI/litgpt#1106]
Our finetuning tutorials end with suggesting to run `litgpt generate` but no mention of `litgpt chat`. I believe chatting is a more intuitive suggestion to immediately use and test the model. So I propose to incluce this as the first step after finetuning in our tutorials.

**Comment by rasbt** [comment]
I agree. I think `generate.py` is more useful when you want to execute the end-to-end example in a bash script, but for the general tutorials let's use `chat.py`
